chains/irisnet/handlerIrisnet.go

```diff
@@ -643,6 +682,8 @@ func (c *P2PConnection) stopPongTimer() {
 // ---------------------- SPAM FILTER INTERFACE --------------------------------
 
 // RunSpamFilter serves as the entry point for a TM Core handler when serving as a spamfilter
+// This function also acts as filter at the very begining of the TM Core, spam filter depends
```

Comment:
wrong!. spamfilter is completely different mode. Number of spam filter is wrong. it is number of worker goroutines spawned